_shows/20_21/2021.md

```diff
@@ -0,0 +1,53 @@
+---
+# Check out the Show Docs for more information
+# -- https://history.newtheatre.org.uk/docs/show/
+# Remove or comment any lines that are not being used
+
+title: "2021"
+playwright: Charlotte Cox
+student_written: true
+period: Autumn
+season: Online
+season_sort: 480
+date_start: 2021-05-05
+date_end: 2021-05-05
+
+cast:
+- role: Alex Bradbury
+  name: Sunenna Sohal
+- role: Government Official 
+  name: Megan Smith
+- role: Tech Company Representative
+  name: Rachel Coussins
+- role: News Reporter
+  name: Olly O'Regan
+  
+crew: # Omit certain roles as required
+- role: Co-Director
+  name: Jasmine Butler
+- role: Co-Director
+  name: Kishan Ganatra
+- role: Producer
+  name: Melina Williams
+- role: Assistant Director
+  name: Geethika Ayiliath
+- role: Publicity Assistant
+  name: Geethika Ayiliath
+- role: Assistant Producer
+  name: Laura Sherratt
+- role: Creative Assistant
+  name: Lelah Gorgin
+- role: Publicity Assistant
+  name: Helena Hunt
+- role: Poster Designer
+  name: Skylar Turnbull Hurd
+- role: Video Editor
+  name: Charlie Finney
+- role: Sound Designer
+  name: Jacob Dean
+
+---
+Do you trust the government to tell you everything? Do you feel as though you are being watched, even in the comfort of your own home? Dystopia concerned themselves with the future, and we are that future. Watch Alex Bradbury tackle modern-day privacy issues in his latest episode of ‘Brave New World’, as he uncovers the truth about yet another exposed corporate lie. NDAs and Confidentiality Clauses can only hide the truth to an extent, and as Alex ventures deeper into the issue of the new TJE phone update, they discover that all is not as discussed in the media, nor released to the public. What has been framed to be a step forward in personal security and privacy measures for all phone users, reveals a far more suspicious story concerning contracted data protection leaks in collusion with the government.![image](https://user-images.githubusercontent.com/92263860/145696189-4ca7a6d1-ff89-4fc7-b599-8410baad306b.png)
+
```

Comment:
Can we look at what these images should be doing please?
(Potentially @BenWoodford76 here)

Reply:
I'm not completely sure as to why it started coming up as that

Reply:
it appears to be a boxed text of the beginning of the show description.

I'm gonna say that it can disappear